src/features/modals/CreateToken/CreateTokenBody/TokenSummary/TokenSummaryField/TokenSummaryField.tsx

```diff
@@ -0,0 +1,24 @@
+//- React Imports
+import { FC } from 'react';
+
+//- Style Imports
+import styles from './TokenSummaryField.module.scss';
+import classnames from 'classnames';
+const cx = classnames.bind(styles);
+
+interface TokenSummaryFieldProps {
+	className?: string;
+	label: string;
+	value: string | number;
+}
+
+export const TokenSummaryField: FC<TokenSummaryFieldProps> = ({
+	className = '',
+	label,
+	value,
+}) => (
+	<div className={cx(styles.TokenSummaryField, className)}>
```

Comment:
You only need `classnames.bind` when you're adding some conditional module classes, e.g.
```
{ TokenSummaryField: true, HasLabel: label.length }
```
You should be able to remove the `cx` definition and use `classnames` directly.